### PR TITLE
[User Roles] Let non-managers see org setting (but disable form)

### PR DESCRIPTION
**Pull request by malted**
Disables all form fields for members

image

### CODE REVIEW

File: app/policies/event_policy.rb [hunk 1]
```diff
@@ -165,12 +165,12 @@ def validate_slug?
     admin_or_user?
   end
 
+  private
+
   def admin_or_user?
```
Comment (garyhtou) [inline app/policies/event_policy.rb]: https://github.com/hackclub/bank/blob/34aecab447b76773f0e91fc128350c8530cb55af/app/controllers/stripe_cards_controller.rb#L69 and https://github.com/hackclub/bank/blob/34aecab447b76773f0e91fc128350c8530cb55af/app/controllers/stripe_cards_controller.rb#L74 will break if this is private!!

File: app/policies/event_policy.rb [hunk 2]
```diff
@@ -33,7 +33,7 @@ def by_airtable_id?
   end
 
   def edit?
-    admin_or_manager?
+    admin_or_user?
```
Comment (garyhtou) [inline app/policies/event_policy.rb]: This policy is good. However, on the page, a member can still type into some fields. Please double check that everything's disabled, including form fields

**Review by garyhtou — CHANGES_REQUESTED** [review]

File: app/policies/event_policy.rb (hunk 1 above)
Comment (garyhtou) [reply app/policies/event_policy.rb]: Fixed by https://github.com/hackclub/hcb/pull/5670

File: app/policies/event_policy.rb (hunk 2 above)
Comment (malted) [reply app/policies/event_policy.rb]: Everything is disabled!

File: app/views/events/_form.html.erb [hunk 3]
```diff
@@ -19,10 +19,13 @@
       <% if @event.logo.attached? %>
         <%= image_tag @event.logo, height: 72 %>
       <% end %>
-      <div class="field--fileupload">
-        <%= form.label :logo, "Choose File", class: "field--fileupload__label" %>
-        <%= form.file_field :logo, accept: "image/*,image/heic", class: "field--fileupload__field" %>
-      </div>
+
+      <% if policy(@event).remove_logo? %>
```
Comment (garyhtou) [inline app/views/events/_form.html.erb]:
```suggestion
      <% if policy(@event).update? %>
```

File: app/views/events/_form.html.erb [hunk 4]
```diff
@@ -37,10 +40,13 @@
       <% if @event.background_image.attached? %>
         <%= image_tag @event.background_image, width: 256 %>
       <% end %>
-      <div class="field--fileupload">
-        <%= form.label :background_image, "Choose File", class: "field--fileupload__label" %>
-        <%= form.file_field :background_image, accept: "image/*,image/heic", class: "field--fileupload__field" %>
-      </div>
+
+      <% if policy(@event).remove_background_image? %>
```
Comment (garyhtou) [inline app/views/events/_form.html.erb]:
```suggestion
      <% if policy(@event).update? %>
```

**Review by garyhtou — APPROVED** [review]

**Pull request merged**